src/EditorFeatures/Core/Options/LegacyGlobalOptionsWorkspaceService.cs

```diff
@@ -22,5 +23,11 @@ public LegacyGlobalOptionsWorkspaceService(IGlobalOptionService globalOptions)
         {
             GlobalOptions = globalOptions;
         }
+
+        public bool RazorUseTabs
+            => GlobalOptions.GetOption(RazorLineFormattingOptionsStorage.UseTabs);
+
+        public int RazorTabSize
+            => GlobalOptions.GetOption(RazorLineFormattingOptionsStorage.TabSize);
```

Comment:
Couldn't this vary from one Razor document to another?

Reply:
My understanding is that it can't.

Reply:
@NTaylorMullen to confirm.

Reply:
I don't believe so unless we have the support for editorconfig /cc @allisonchou